browser.js

```diff
@@ -8,7 +8,8 @@ var debug = util.debuglog('net');
 var proxy = {
 	protocol: (window.location.protocol == 'https:') ? 'wss' : 'ws',
 	hostname: window.location.hostname,
-	port: window.location.port
+	port: window.location.port,
+        url: '/api/vm/net'
```

Comment:
A more accurate name would be `path` - `url` contains `protocol`, `hostname` and `port`.